lib/game_teams_manager.rb

```diff
@@ -59,4 +100,35 @@ def add_wins_losses(gt_results, coach_record_start)
     end
     coach_record_start
   end
+
+  def initialize_shots_and_goals_per_team(gt_results)
```

Comment:
Okay, this is probably like way overly nitpicky. I'm just thinking about using the term "initialize" in a method which isn't initialized upon creation of the class. For whatever reason it feels off to me? Not a huge deal, just a thought to share!